caikit/config/config.py

```diff
@@ -0,0 +1,152 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Config methods for the `caikit` library. Mainly interacts with `config.yml`.
+"""
+
+# Standard
+from typing import Optional
+import os
+import threading
+
+# First Party
+import aconfig
+import alog
+
+# Local
+from caikit.core.toolkit.errors import error_handler
```

Comment:
yeah, that's just in there because of the `error_handler` awkwardness that I'll go get rid of. It should probably just read from config instead of having separate globals that folks can mess with that then need to be set here

Reply:
Question about `caikit.config` vs. `caikit.core.config` still stands but we can kill the import circle